supplier_product/tests.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from supplier_product.models import SupplierProduct
+
+
+@pytest.mark.django_db()
+class TestSupProduct:
+
+    def test_save_supplier_product(self, supplier_product0):
+        assert supplier_product0 not in SupplierProduct.objects.all()
+        SupplierProduct.save_sup_product(supplier_product0)
+        assert supplier_product0 in SupplierProduct.objects.all()
+
+    def test_delete_supplier_product(self, supplier_product0):
+        assert supplier_product0 not in SupplierProduct.objects.all()
+        SupplierProduct.save_sup_product(supplier_product0)
+        assert supplier_product0 in SupplierProduct.objects.all()
+        supplier_product0.delete()
+        assert supplier_product0 not in SupplierProduct.objects.all()
+
+    def test_increase_supp_product_quantiy(self, supplier_product0):
+        new_quantity = supplier_product0.quantity + 1
+        old_quantity = supplier_product0.quantity
+        supplier_product0.increase_sup_product_quantity(new_quantity)
+        assert supplier_product0.quantity == old_quantity + new_quantity
+
+    def test_decrease_supp_product_quantiy(self, supplier_product0):
```

Comment:
please follow the original method's name (AKA `sup` instead of `supp`, `quantity` instead of `quantiy`